jetty-server/src/test/java/org/eclipse/jetty/server/handler/InetAccessHandlerTest.java

```diff
@@ -136,39 +156,68 @@ public void testHandler(String include, String exclude, String includeConnectors
+
+                // test simple filters 2 - 5
+                {"127.0.0.1", "", "", "", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "", "", "200;200"},
+                {"192.0.0.1", "", "", "", "403;403"},
+                {"192.0.0.1-192.0.0.254", "", "", "", "403;403"},
+
+                // 6 - 9 - test includeConnector
+                {"127.0.0.1", "", "http_connector1", "", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "http_connector1", "", "200;200"},
+                {"192.0.0.1", "", "http_connector1", "", "403;200"},
+                {"192.0.0.1-192.0.0.254", "", "http_connector1", "", "403;200"},
+                {"192.0.0.1", "", "http_connector2", "", "200;403"},
+                {"192.0.0.1-192.0.0.254", "", "http_connector2", "", "200;403"},
+
+                // 10 - 13 - test includeConnector names where none of them match
+                {"127.0.0.1", "", "nothttp", "", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "nothttp", "", "200;200"},
+                {"192.0.0.1", "", "nothttp", "", "200;200"},
+                {"192.0.0.1-192.0.0.254", "", "nothttp", "", "200;200"},
+
+                // 14 - 17 - text excludeConnector
+                {"127.0.0.1", "", "", "http_connector1", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "", "http_connector1", "200;200"},
+                {"192.0.0.1", "", "", "http_connector1", "200;403"},
+                {"192.0.0.1-192.0.0.254", "", "", "http_connector1", "200;403"},
+                {"192.0.0.1", "", "", "http_connector2", "403;200"},
+                {"192.0.0.1-192.0.0.254", "", "", "http_connector2", "403;200"},
+
+                // 18 - 21 - test excludeConnector where none of them match.
+                {"127.0.0.1", "", "", "nothttp", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "", "nothttp", "200;200"},
+                {"192.0.0.1", "", "", "nothttp", "403;403"},
+                {"192.0.0.1-192.0.0.254", "", "", "nothttp", "403;403"},
+
+                // 22 - 25 - both connectors are excluded
+                {"127.0.0.1", "", "", "http_connector1;http_connector2", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "", "http_connector1;http_connector2", "200;200"},
+                {"192.0.0.1", "", "", "http_connector1;http_connector2", "200;200"},
+                {"192.0.0.1-192.0.0.254", "", "", "http_connector1;http_connector2", "200;200"},
+
+                // 26 - 29 - both connectors are included
+                {"127.0.0.1", "", "http_connector1;http_connector2", "", "200;200"},
+                {"127.0.0.1-127.0.0.254", "", "http_connector1;http_connector2", "", "200;200"},
+                {"192.0.0.1", "", "http_connector1;http_connector2", "", "403;403"},
+                {"192.0.0.1-192.0.0.254", "", "http_connector1;http_connector2", "", "403;403"},
+                
+                // 30 - 33 - exclude takes precedence over include
```

Comment:
Numbering is nice, but I hope more tests aren't added in the future, because it's unlikely that the committer will remember to update the numbering.